src/ui_ng/lib/src/tag/tag.component.ts

```diff
@@ -514,6 +520,13 @@ export class TagComponent implements OnInit, AfterViewInit {
                       findedList = BathInfoChanges(findedList, res);
                     });
               }).catch(error => {
+            if (error.status === 503) {
+              Observable.forkJoin(this.translateService.get('BATCH.DELETED_FAILURE'),
+                  this.translateService.get('REPOSITORY.TAGS_NO_DELETE')).subscribe(res => {
+                findedList = BathInfoChanges(findedList, res[0], false, true, res[1]);
```

Comment:
typo in the variable name `BatchInfoChanges`?

Reply:
The function name was setted BathInfoChanges at the beginning of time,   if changes , may a lot of pages should modiy for the variable name .